server/aws-lsp-codewhisperer/src/language-server/telemetryService.ts

```diff
@@ -185,6 +191,16 @@ export class TelemetryService extends CodeWhispererServiceToken {
         if (options?.conversationId === undefined) {
             return
         }
+        if (this.enableTelemetryEventsToDestination) {
```

Comment:
Let's add a unit test for this condition

Reply:
There is already a unit test update for this when I am setting telemetryService.updateTelemetryEventsToDestination to true.